guake/tests/test_guake.py

```diff
@@ -180,3 +180,11 @@ def test_guake_save_tabs_and_restore(mocker, g, fs):
     assert nb.get_n_pages() == 3
     assert nb.get_tab_text_index(1) == "foobar"
     assert nb.get_tab_text_index(2) == "python"
+
+
+def test_guake_hide_tabbar_if_one_tab(mocker, g, fs):
+    # Set hide-tabs-if-one-tab to True
+    mocker.patch.object(g.settings.general, "get_boolean", return_value=True)
```

Comment:
Does this line patch all general option to True?

Reply:
yeah, but the tab option should be the only relevant one, I just went by what the test case above this one was doing.